docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,27 @@
 # Parameters
 
-!!! warning
+## ROS parameters
 
-    Under Construction
+The ROS packages in Autoware may have ROS parameter(s) for setting some variables that should be customized depending on the applications.
+It is recommended not to set default values when you declare ROS parameters to avoid unintended behaviors.
```

Comment:
You mean about a line `It is recommended not to set default values when you declare ROS parameters to avoid unintended behaviors.`, right?

I could not find any part that refers to "avoid using default parameters" in ROS 2 coding guidelines, so I feel it's better to leave it just as a recommendation for now.
(Or maybe we can just remove this line, since I cannot correctly recall if I had enough discussion about this recommentation...)

Reply:
@kenji-miyake 
> It's an Autoware's guideline, proposed by TIER IV.

I see, I didn't know that

> It's also okay for me. It's better to create a discussion thread and investigate more.

OK, I also agree that rules can be determined later after some discussions on AWF. So I will leave the PR as it is for now.

Reply:
@kminoda How about this?

```suggestion
It is recommended not to set default values when declaring ROS parameters to avoid unintended behaviors due to accidental use of default values.
Instead, set parameters from configuration files named `*.param.yaml`.
```

Reply:
Thank you! I take that idea.
Resolved: https://github.com/autowarefoundation/autoware-documentation/pull/271/commits/843889d8e7a65d7ee4d5cd42633625a86f9948aa